### PR TITLE
Add gpuci/rapidsai and gpuci/rapidsai-driver for arm64

### PR DESCRIPTION
**Pull request by jjacobelli**

### CODE REVIEW

**Comment by jjacobelli** [comment]
rerun tests

File: rapidsai/devel.arm64.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      -c gpuci \
+      -c rapidsai-nightly \
+      cudatoolkit=${CUDA_VER} \
+      git \
+      git-lfs \
+      gpuci-tools \
+      jq \
+      libgcc-ng=${BUILD_STACK_VER} \
+      libstdcxx-ng=${BUILD_STACK_VER} \
+      mamba \
```
Comment (raydouglass) [inline rapidsai/devel.arm64.Dockerfile]: mamba should go in the base environment

File: rapidsai/devel.arm64.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+RUN gpuci_conda_retry create --no-default-packages --override-channels -n rapids \
+      -c nvidia \
+      -c conda-forge \
+      -c gpuci \
+      -c rapidsai-nightly \
+      cudatoolkit=${CUDA_VER} \
+      git \
+      git-lfs \
+      gpuci-tools \
+      jq \
```
Comment (raydouglass) [inline rapidsai/devel.arm64.Dockerfile]: jq is installed in the base env so I don't think it is needed here

**Review by ajschmidt8 — APPROVED** [review]

**Review by raydouglass — CHANGES_REQUESTED** [review]

File: rapidsai/devel.arm64.Dockerfile (hunk 1 above)
Comment (jjacobelli) [reply rapidsai/devel.arm64.Dockerfile]: I used the `devel.Dockerfile` as base and we are installing `mamba` in the `rapids` env. Should I move it to the `base` env?

File: rapidsai/devel.arm64.Dockerfile (hunk 2 above)
Comment (jjacobelli) [reply rapidsai/devel.arm64.Dockerfile]: Done

File: rapidsai/devel.arm64.Dockerfile (hunk 1 above)
Comment (raydouglass) [reply rapidsai/devel.arm64.Dockerfile]: Yes, I think so

Comment (jjacobelli) [reply rapidsai/devel.arm64.Dockerfile]: Done

**Comment by jjacobelli** [comment]
rerun tests

**Comment by jjacobelli** [comment]
rerun tests

**Comment by jjacobelli** [comment]
rerun tests

**Review by raydouglass — APPROVED** [review]

**Pull request merged**